app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-input.tsx

```diff
@@ -17,13 +17,22 @@ interface PersonalKeyInputProps {
   onChange?: (nextValue: string) => void;
 }
 
+/**
+ * Normalize an input value for validation comparison.
+ *
+ * @param string Denormalized value.
+ *
+ * @return Normalized value.
+ */
+const normalize = (string: string) => string.toLowerCase().replace(/o/g, '0').replace(/[il]/g, '1');
```

Comment:
yeah it's silly to have a whole library that we can replicate with two regexes, this seems easy enough to maintain 👍